test/image/mocks/gl3d_surface-with-no-z-data.json

```diff
@@ -0,0 +1,33 @@
+{
+    "data": [
```

Comment:
Could we turn these test cases into jasmine tests instead? This `surface_test.js` block 

https://github.com/plotly/plotly.js/blob/997e720c8f40fda68b98e4c365baf1ac62e84cfd/test/jasmine/tests/surface_test.js#L9

should be a nice place to test the default logic.